src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php

```diff
@@ -1,30 +1,34 @@
-<div class="CMSInterfacePopup">
-    <form name="indexer" method="post" action="<?=PATH_CMS_CONTROLLER; ?>" accept-charset="UTF-8">
-        <input type="hidden" name="pagedef" value="<?=TGlobal::OutHTML($data['pagedef']); ?>"/>
-        <input type="hidden" name="_pagedefType" value="<?= $data['_pagedefType']; ?>"/>
-        <input type="hidden" name="module_fnc[<?=TGlobal::OutHTML($data['sModuleSpotName']); ?>]"
-               value="TickerIndexGeneration"/>
+<div class="card CMSInterfacePopup">
+    <div class="card-header">
+        Article search index
```

Comment:
Could you add this text to translations and use the translated strings? Also please name it "Product search index" as we try to use the term "product" consistently.

Reply:
OK, done.